pkgs/development/libraries/ctranslate2/default.nix

```diff
@@ -87,5 +87,6 @@ stdenv.mkDerivation rec {
     changelog = "https://github.com/OpenNMT/CTranslate2/blob/${src.rev}/CHANGELOG.md";
     license = licenses.mit;
     maintainers = with maintainers; [ hexa misuzu ];
+    broken = !(withCuDNN -> withCUDA);
```

Comment:
`broken = false` when we manually pass `withCUDA = withCuDNN = true` and `cudaPackages_10`?

Reply:
`ctranslate` uses CUDA redistributables and doesn't appear to support the runfile installer, so it won't eval pre-11.4. I'll add a guard for that

Reply:
Resolved in latest; shouldn't break eval now.